visidata/movement.py

```diff
@@ -114,7 +101,6 @@ def visibleWidth(self):
 Sheet.addCommand(None, 'go-bottom', 'sheet.cursorRowIndex = len(rows); sheet.topRowIndex = cursorRowIndex-nScreenRows', 'go all the way to the bottom of sheet')
 Sheet.addCommand(None, 'go-rightmost', 'sheet.leftVisibleColIndex = len(visibleCols)-1; pageLeft(); sheet.cursorVisibleColIndex = len(visibleCols)-1', 'go all the way to the right of sheet')
 
-Sheet.addCommand('c', 'go-col-regex', 'sheet.cursorVisibleColIndex=nextColRegex(inputRegex("column name regex: ", type="regex-col", defaultLast=True))', 'go to next column with name matching regex')
 Sheet.addCommand('zc', 'go-col-number', 'sheet.cursorVisibleColIndex = int(input("move to column number: "))', 'go to given column number (0-based)')
```

Comment:
I noticed in your commit message that you mention this one was being moved, too. Did you intend to move it?

Reply:
At first I meant to move it. Later I decided against it and forgot to change the message. I figured it belonged next to `go-row-number`, in `movement.py`. Now that I see your menu changes, the move seems better after all. Putting it in `go_col.py` keeps it with the group of commands for `Menu> Column > Goto`.

Could you squash these last few commits for the move? Thank you.